fdbclient/NativeAPI.actor.cpp

```diff
@@ -265,11 +265,11 @@ void DatabaseContext::getLatestCommitVersions(const Reference<LocationInfo>& loc
 void updateCachedReadVersionShared(double t, Version v, DatabaseSharedState* p) {
 	MutexHolder mutex(p->mutexLock);
 	if (v >= p->grvCacheSpace.cachedReadVersion) {
-		TraceEvent(SevDebug, "CacheReadVersionUpdate")
-		    .detail("Version", v)
-		    .detail("CurTime", t)
-		    .detail("LastVersion", p->grvCacheSpace.cachedReadVersion)
-		    .detail("LastTime", p->grvCacheSpace.lastGrvTime);
+		//TraceEvent(SevDebug, "CacheReadVersionUpdate")
```

Comment:
I take it from this that we are logging `SevDebug` events in simulation. Would it make sense to log this at an even lower severity, like `SevVerbose`? I'm ok with commenting it out, if you prefer that instead.

Reply:
I saw some test runs where this was responsible for more than half of all the trace lines that were produced over the course of a simulation run. This suggest that this is sometimes running in some tight loop and constructing a `TraceEvent` is not free. So I would rather have this in a commented out state